utils/splunk/splunk_telemetry_base.py

```diff
@@ -231,6 +266,14 @@ def _auth_session(self, instance):
         """ This method is mocked for testing. Do not change its behavior """
         instance.splunkHelper.auth_session()
 
+    def _is_valid_token(self, instance, token):
```

Comment:
Method name differs from what is called on SplunkHelper, consistency?

Reply:
fixed.